fastq_filtrator.py

```diff
@@ -0,0 +1,37 @@
+def read_file(input_fastq):
+    with open(input_fastq) as file:
+        content = file.readlines()
+    reads = []
+    for i in range(0, len(content) - 3, 4):
+        reads.append(list(content[i:i + 4]))
```

Comment:
Преобразование в список излишне
```suggestion
        reads.append(content[i:i + 4])
```